tests/data_files/Makefile

```diff
@@ -48,6 +48,18 @@ test-ca-sha256.crt: $(test_ca_key_file_rsa) $(test_ca_config_file) test-ca.csr
 	$(OPENSSL) req -x509 -config $(test_ca_config_file) -key $(test_ca_key_file_rsa) -passin "pass:$(test_ca_pwd_rsa)" -set_serial 0 -days 3653 -sha256 -in test-ca.csr -out $@
```

Comment:
Don't we want the notbefore and notafter to be set for test-ca.crt, test-ca-sha1.crt and test-ca-sha256.crt?

Reply:
But we do.
In `mbedtls-2.7` the certificates are generated using openssl, as opposed to `mbedtls-2.16` which uses `cert_write`.
in openssl, the notbefore and notafter are generated with the `-days` parameter